### ISSUE TITLE
rsync remote script emits "join: ... is not sorted" in logs

### ISSUE THREAD

**Issue by portante** [distributed-system-analysis/pbench#447]
See lines like this in the logs:

```
join: -:693: is not sorted: EC2::ip-XXX-XX-XX-XXX/wss_scale_140x4_x2_190s_190s___.tar.xz        5416132
```

**Comment by portante** [comment]
Perhaps using `sort -k 1b,1` will fix this?  See http://stackoverflow.com/questions/26626407/join-gives-warning-file1-is-not-in-sorted-order

**Comment by portante** [comment]
Closed by #477.
